docs/contributing/contributing-releases/README.md

```diff
@@ -34,23 +34,28 @@ If this workflow run fails, or if we encounter an issue with an RC release, plea
 
 If sample validation passes, we can start the process of creating the final release.
 
-1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change.
+1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change into the bicep-extensibility branch.
```

Comment:
I think it would be good to explicitly call out release branch as well

```suggestion
1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change into the bicep-extensibility and release/<release-version> branch.
```